### DISCUSSION TITLE
[Wrong device] TS0601_dimmer_knob recognized as TS0601_dimmer_1

### DISCUSSION THREAD

**Discussion by fahdo60** [Koenkk/zigbee2mqtt#22660] [Devices]
Hello

I have this device
https://www.zigbee2mqtt.io/devices/TS0601_dimmer_knob.html

recognized as
https://www.zigbee2mqtt.io/devices/TS0601_dimmer_1.html

Is there a way to force the correct model or should you make a fix?

Thanks

**Comment by Koenkk** [comment]
Could you provide the data/database.db entry of your device?

**Reply by fahdo60** [reply]
Yes

{"id":2,"type":"Router","ieeeAddr":"0x60a423fffefe8520","nwkAddr":4437,"manufId":4098,"manufName":"_TZE200_dfxkcots","powerSource":"Mains (single phase)","modelId":"TS0601","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":81,"inClusterList":[0,4,5,61184],"outClusterList":[25,10],"clusters":{"genBasic":{"attributes":{"65503":"C��-\u0012D��-\u0012\u0000\u0000\u0000\u0000\u0005�\u0012�-\u0012�\u0012�-\u0012","65506":31,"65508":0,"65534":0,"modelId":"TS0601","manufacturerName":"_TZE200_dfxkcots","powerSource":1,"zclVersion":3,"appVersion":66,"stackVersion":0,"hwVersion":1,"dateCode":""}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x90395efffe3414f3","endpointID":1},{"cluster":8,"type":"endpoint","deviceIeeeAddress":"0x90395efffe3414f3","endpointID":1}],"configuredReportings":[],"meta":{}}},"appVersion":66,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{"configured":-24894193},"lastSeen":1715985796768}

**Reply by Koenkk** [reply]
@timota since you have a `_TZE200_dfxkcots`, can you confirm it should be detected as TS0601_dimmer_knob?

**Reply by timota** [reply]
Hi @Koenkk. 
On my opinion, it should not be detected as a knob, as it does not have any rotary elements., at least if rotary knob behaves differently from `+/-` buttons.

In case if you need db entry 
```json
{
  "id": 6,
  "type": "Router",
  "ieeeAddr": "0x60a423fffe61bfec",
  "nwkAddr": 64152,
  "manufId": 4098,
  "manufName": "_TZE200_dfxkcots",
  "powerSource": "Mains (single phase)",
  "modelId": "TS0601",
  "epList": [
    1
  ],
  "endpoints": {
    "1": {
      "profId": 260,
      "epId": 1,
      "devId": 81,
      "inClusterList": [
        0,
        4,
        5,
        61184
      ],
      "outClusterList": [
        25,
        10
      ],
      "clusters": {
        "genBasic": {
          "attributes": {
            "65503": "���-\u0012���-\u0012",
            "65506": 31,
            "65508": 0,
            "65534": 0,
            "modelId": "TS0601",
            "manufacturerName": "_TZE200_dfxkcots",
            "stackVersion": 0,
            "dateCode": "",
            "zclVersion": 3,
            "appVersion": 66,
            "powerSource": 1
          }
        }
      },
      "binds": [
        {
          "cluster": 6,
          "type": "endpoint",
          "deviceIeeeAddress": "0x00124b0026d132c9",
          "endpointID": 1
        }
      ],
      "configuredReportings": [],
      "meta": {}
    }
  },
  "appVersion": 66,
  "stackVersion": 0,
  "hwVersion": 1,
  "dateCode": "",
  "zclVersion": 3,
  "interviewCompleted": true,
  "meta": {
    "reporting": 1,
    "configured": -903893911
  },
  "lastSeen": 1716201934017
}
```

**Reply by fahdo60** [reply]
the device that I acquired is indeed the rotary moes button - / + to decrease or increase the light.
and you can press it to turn it on or off.

**Reply by fahdo60** [reply]
I migrated from smartlife to homme assistant always for this device, for the "light type" option I could choose between halogen incandescent LED. but the led option resets automatically
